### DISCUSSION TITLE
Bimodal analysis

### DISCUSSION THREAD

**Discussion by emjblake** [nloyfer/wgbs_tools#57] [Q&A] [answered]
What does `BPI` in the `test_bimodal` function stand for? 

```
wgbstools test_bimodal -r $region Left_Ventricle_STL001.IGF2.pat.gz
LL: -1401.2486342579095 | 281 reads | 1437 observed | BPI: 0.5087178153029501
LL: -457.82607811961105 | 281 reads | 1437 observed | BPI: 0.8018590355733447
pvalue: 0.000e+00
```

**Comment by yonniejon** [comment, marked as answer]
Indeed. It's the average likelihood (geometric mean) per observation (CpG site). We will have to find a better name....

**Reply by emjblake** [reply]
@yonniejon, to confirm, the output is single model (first line) vs two model (second line), where pvalue threshold <=0.05 means bimodal distribution? Thanks!

**Reply by yonniejon** [reply]
Yes, you are correct
